vitest.shims.d.ts

```diff
@@ -0,0 +1,3 @@
+/// <reference types="@types/chai-dom" />
```

Comment:
Maybe this file should be added to the .gitignore?

Reply:
Why?